medical/data/ir_sequence_data.xml

```diff
@@ -13,4 +13,4 @@
         </record>
 
     </data>
-</odoo>
+</odoo>
```

Comment:
All files should end with a newline to meet POSIX file standard - http://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline